### PR TITLE
Marc

### PR DESCRIPTION
**Pull request by drvinceknight**
Opening this to discuss/view etc...

### CODE REVIEW

File: tex/paper.tex [hunk 1]
```diff
@@ -36,17 +36,36 @@
...
+original source code available (Axelrod's second tournament, in FORTRAN); in no
+cases is the available code well-documented, easily modifiable, or released
+with significant test suites.
+
+To make matters more complicated, often a newly-created strategy is studied in
+isolation by opponents chosen by the strategy's creators, and often such
+strategies are not sufficiently described to enable reliable recreation
+(in the absense of source code), with \cite{slany2007some} being a notable
+counter-example. In some cases strategies are revised without updates to their
+names or published implementations \cite{li2007design} \cite{li2011engineering}.
```
Comment (drvinceknight) [inline tex/paper.tex:53]: \cite{li2007design, li2011engineering}

(Not necessarily telling anyone to do this, just making a note).

File: tex/paper.tex [hunk 2]
```diff
@@ -120,10 +141,11 @@ \subsection{Review of the literature}\label{sec:review}
             Year     & Reference           & Number of Strategies & Type     & Source Code\\
             \midrule
             1979     & \cite{Axelrod1980a} & 13                   & Standard & Not immediately available\\
-            1979     & \cite{Axelrod1980b} & 64                   & Standard & Not immediately available\\
+            1979     & \cite{Axelrod1980b} & 64                   & Standard & Available in FORTRAN\\
             1991     & \cite{Bendor1991}   & 13                   & Noisy    & Not immediately available\\
             2002     & \cite{Stephens2002} & 16                   & Wildlife & Not applicable\\
-            % Is this worth including? If so I think we need more.
+            2005     & \cite{kendall2007iterated} & 223           & Varied   & Not available \\
+            2012     & \cite{Stewart2012} & 13                    & Standard & Not fully available \\
```
Comment (drvinceknight) [inline tex/paper.tex:148]: Based on what you said in intro, should slany2007some be here? Haven't read the paper.

Comment (drvinceknight) [inline tex/paper.tex:148]: Have seen the paper: I see that it shouldn't.

Comment (marcharper) [inline tex/paper.tex:148]: It refers to the 2004/5 tournaments

**Comment by marcharper** [comment]
I'll keep adding to my branch as I have time but feel free to merge and make edits.

**Comment by drvinceknight** [comment]
Cool, if I do any work I'll be on a branch also.

Vince

**Comment by drvinceknight** [comment]
I'm going to go ahead and merge this in as I want to work from it on my own branch.

**Pull request merged**